src/app/clusters/network-commissioning/network-commissioning.h

```diff
@@ -59,8 +59,8 @@ class Instance : public CommandHandlerInterface,
 
     // CommandHandlerInterface
     void InvokeCommand(HandlerContext & ctx) override;
-    CHIP_ERROR EnumerateAcceptedCommands(const ConcreteClusterPath & cluster, CommandIdCallback callback, void * context) override;
-    CHIP_ERROR EnumerateGeneratedCommands(const ConcreteClusterPath & cluster, CommandIdCallback callback, void * context) override;
+    bool AcceptsCommandId(const ConcreteCommandPath & commandPath) override;
+    bool GeneratesCommandId(const ConcreteCommandPath & commandPath) override;
 
```

Comment:
This breaks the ability of CHI to not require the ZAP metadata to include a command. The AcceptedCommandList should not need to be exhaustive and include commands not generated.  Otherwise we are coupling MORE, and also requiring that ZAP metadata includes all commands, when it was not required before.

Reply:
This is a fair concern. Keeping this comment open as a blocker for the PR to see what we do here.

Ideally I would like CHI to be fully stand alone then, but then we need some interface that is similar to `DataModel::Provider` capabilities (i.e. return iteration AND metadata). Still need to determine what to do here.